.github/workflows/generate_third_party_licenses.yml

```diff
@@ -0,0 +1,10 @@
+name: Generate THIRD-PARTY-LICENSES
+
```

Comment:
Sorry for missing this previously, but isn't this working if we simply add this in `github/workflows/generate_third_party_licenses_workflow.yml` and remove this file? 

```
push:
    tags: [ "v[0-9]+.[0-9]+.[0-9]+" ]
```

Reply:
We are generating 3rd party license file in two place at the moment. One action is the build of wheels for  x86, the second action create only license file. The second action is needed that we could start build of arm wheels before all the x86 wheels would be build.

Reply:
I understood that, what I meant is does it work to have something like in the next file and remove this one? 
```
on:
  push:
    tags: [ "v[0-9]+.[0-9]+.[0-9]+" ]
  workflow_call:
```